### PR TITLE
Add post WG

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: _posts/2021-01-15-galaxy-wg.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'New Galaxy Community Organization: Working Groups'
+tags: [galaxy]
+---
+
+<br>
```
Comment (bgruening) [inline _posts/2021-01-15-galaxy-wg.md:7]: Is see this br more often what is the reason?

File: _posts/2021-01-15-galaxy-wg.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+The Galaxy community is undergoing an __organizational transformation__, with the main focus on improving our global coordination in different areas by defining [__Working Groups__](https://galaxyproject.org/community/wg/). 
+
+The Working Groups are __distributed across the globe__ and diverse in the topics covered  (admin, backend, UI/UX...). Although their initial definition is set, they might be refined over time.
+
+Each working group has the __autonomy__ to organize themselves internally and it's responsible for the __definition of goals__ in each quarter of the year. The goals, together with the input from the community, will be incorporated into the __community roadmap__.
```
Comment (bgruening) [inline _posts/2021-01-15-galaxy-wg.md:12]: communit and steering committee?

File: _posts/2021-01-15-galaxy-wg.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'New Galaxy Community Organization: Working Groups'
+tags: [galaxy]
+---
+
+<br>
+The Galaxy community is undergoing an __organizational transformation__, with the main focus on improving our global coordination in different areas by defining [__Working Groups__](https://galaxyproject.org/community/wg/). 
+
+The Working Groups are __distributed across the globe__ and diverse in the topics covered  (admin, backend, UI/UX...). Although their initial definition is set, they might be refined over time.
```
Comment (bgruening) [inline _posts/2021-01-15-galaxy-wg.md:10]: The working groups are not distributed but the members? The working groups are open for everyone and have members from all over the globe...

File: _posts/2021-01-15-galaxy-wg.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'New Galaxy Community Organization: Working Groups'
+tags: [galaxy]
+---
+
+<br>
+The Galaxy community is undergoing an __organizational transformation__, with the main focus on improving our global coordination in different areas by defining [__Working Groups__](https://galaxyproject.org/community/wg/).
```
Comment (bgruening) [inline _posts/2021-01-15-galaxy-wg.md:8]: Maybe we can add a motivation why this is needed... Growing community, many areas with different expertise in the community ...

File: _posts/2021-01-15-galaxy-wg.md (hunk 1 above)
Comment (beatrizserrano) [reply _posts/2021-01-15-galaxy-wg.md:7]: the space between the header and the text does not seem enough to me :)

Comment (bgruening) [reply _posts/2021-01-15-galaxy-wg.md:7]: I guess then we should tweak the CSS

**Comment by beatrizserrano** [comment]
I would love to have your input here, @tnabtaf :)

File: _posts/2021-01-15-galaxy-wg.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'New Galaxy Community Organization: Working Groups'
+tags: [galaxy]
+---
+
+<br>
+The Galaxy community is undergoing an __organizational transformation__, with the main focus on improving the global coordination in different areas by defining [__Working Groups__](https://galaxyproject.org/community/wg/). The motivations behind this shift are the rapid growth of the community and its diversity.
+
+The Working Groups are open to everyone, with members __distributed across the globe__, and assorted in the topics covered  (admin, backend, UI/UX...). Although their initial definition is set, they might be refined over time.
```
Comment (tnabtaf) [inline _posts/2021-01-15-galaxy-wg.md:10]: Although their initial definition is set, they might be refined over time.

&rarr;

The set of working groups and their missions will evolve over time.

File: _posts/2021-01-15-galaxy-wg.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+Each working group has the __autonomy__ to organize themselves internally and it's responsible for the __definition of goals__ in each quarter of the year. Those goals, together with the input from the community and the steering committee, will be incorporated into the __community roadmap__.
```
Comment (tnabtaf) [inline _posts/2021-01-15-galaxy-wg.md:12]: it's &rarr; is
definition of goals &rarr; defining goals

File: _posts/2021-01-15-galaxy-wg.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+The Working Groups provide __entry points__ into the community for new members, will __improve the communication__ between experts and will help with __channelling the contributions__. 
+
+This new organization has been discussed in two Developer Roundtables (second recording soon to be available) and the ELIXIR-Galaxy community has been informed in the monthly meetings.
```
Comment (tnabtaf) [inline _posts/2021-01-15-galaxy-wg.md:16]: Working on getting that 2nd video...

**Review by tnabtaf — COMMENTED** [review]
Looks good.  Some small suggested edits.

**Comment by beatrizserrano** [comment]
Thanks a lot, @tnabtaf 💯

**Pull request merged**